src/test/java/api/support/APITests.java

```diff
@@ -278,6 +285,7 @@ public void afterEach() {
     patronNoticesClient.deleteAll();
     scheduledNoticesClient.deleteAll();
     patronSessionRecordsClient.deleteAllIndividually();
+    templateClient.deleteAll();
```

Comment:
What is the difference between `templateFixture.deleteAll` and `templateClient.deleteAll`?

Reply:
There is no difference, you asked me use tempalteFixture above. In my vision that it should contain next method: create, delete, deleteAll.

Reply:
templateClient.deleteAll removed